samples_project/Assets/SampleViewer/Samples/FeatureLayer/FeatureLayer.cs

```diff
@@ -0,0 +1,421 @@
+        {
+            if (GetAllFeatures)
+            {
+                CreateAllFeatures();
+            }
+            else
+            {
+                CreateFeatureRange();
+            }
+            
+            if (UIManager.DisplayText != FeatureLayerUIManager.TextToDisplay.LinkError
+                && UIManager.DisplayText != FeatureLayerUIManager.TextToDisplay.IndexOutOfBoundsError)
+            {
+                UIManager.DisplayText = FeatureLayerUIManager.TextToDisplay.Information;
+            }
+        }
+        else
+        {
+            UIManager.DisplayText = FeatureLayerUIManager.TextToDisplay.CoordinatesError;
+        }
+    }
+
+    private void CreateAllFeatures()
+    {
+        foreach (var feature in jFeatures) 
+        {
+            Feature currentFeature = new Feature();
+            var featureItem = Instantiate(featurePrefab, this.transform);
+            featureItem.tag = "FeatureItem";
+            featureItem.layer = 7;
+            featureInfo = featureItem.GetComponent<FeatureData>();
+            locationComponent = featureItem.GetComponent<ArcGISLocationComponent>();
+            var coordinates = feature.SelectToken("geometry").SelectToken("coordinates").ToArray();
+            var properties = feature.SelectToken("properties").ToArray();
+
+            if (GetAllOutfields)
+            {
+                foreach (var value in properties)
+                {
+                    var key = value.ToString();
+                    var props = key.Split(":");
+                    currentFeature.properties.propertyNames.Add(props[0]);
+                    currentFeature.properties.data.Add(props[1]);
+                    featureInfo.Properties.Add(props[1]);
+                }
+            }
+            else
+            {
+                for (var j = 0; j < outfields.Count; j++)
+                {
+                    if (OutfieldsToGet.Contains(outfields[j]))
+                    {
+                        var key = properties[j].ToString();
+                        var props = key.Split(":");
+                        currentFeature.properties.propertyNames.Add(props[0]);
+                        currentFeature.properties.data.Add(props[1]);
+                        featureInfo.Properties.Add(props[1]);
+                    }
+                }
+            }
+
+            foreach (var coordinate in coordinates)
+            {
+                currentFeature.geometry.coordinates.Add(Convert.ToDouble(coordinate));
+                featureInfo.Coordinates.Add(Convert.ToDouble(coordinate));
+            }
+
+            featureInfo.ArcGISCamera = arcGISCamera;
+            ArcGISPoint Position = new ArcGISPoint(featureInfo.Coordinates[0], featureInfo.Coordinates[1],
+                stadiumSpawnHeight, new ArcGISSpatialReference(featureSRWKID));
+            ArcGISRotation Rotation = new ArcGISRotation(0.0, 90.0, 0.0);
```

Comment:
```suggestion
            ArcGISRotation rotation = new ArcGISRotation(0.0, 90.0, 0.0);
```
Correct case for local variables.